source/common/stats/symbol_table_impl.cc

```diff
@@ -12,6 +13,34 @@ namespace Stats {
 static const uint32_t SpilloverMask = 0x80;
 static const uint32_t Low7Bits = 0x7f;
 
+StatName::StatName(const StatName& src, SymbolStorage memory) : size_and_data_(memory) {
+  memcpy(memory, src.size_and_data_, src.size());
+}
+
+std::string StatName::toString(const SymbolTable& table) const {
+  return table.decode(data(), dataSize());
+}
+
+#ifndef ENVOY_CONFIG_COVERAGE
+void StatName::debugPrint() {
+  if (size_and_data_ == nullptr) {
+    std::cout << "Null StatName" << std::endl << std::flush;
```

Comment:
Any reason to not just use normal Envoy logging APIs here?

Reply:
I'll fix all the logging; was being lazy.

Reply:
Actually in this particular place, the intent is use this interactively from the debugger, rather than a running server; should I still do that with normal logging? At info level? I'd definitely want it to come out rather than be filtered.

Reply:
Whatever level you want is fine, though can't you adjust the logging level on the command line to whatever you want before debugging? That's what I typically do.